AlfrescoApp/View Controllers/FileFolder View Controller/BaseCollectionViewFlowLayout.m

```diff
@@ -180,8 +196,16 @@ - (UICollectionViewLayoutAttributes *)initialLayoutAttributesForAppearingItemAtI
     attributes.editing = self.isEditing;
     attributes.isSelectedInEditMode = NO;
     attributes.thumbnailWidth = self.thumbnailWidth;
-    attributes.shouldShowSeparatorView = (self.numberOfColumns == 1) ? YES : NO;
-    attributes.shouldShowAccessoryView = (self.numberOfColumns == 1) ? YES : NO;
+    attributes.shouldShowSeparatorView = (self.numberOfColumns == 1);
+    attributes.shouldShowAccessoryView = (self.numberOfColumns == 1);
+    attributes.shouldShowNodeDetails = (self.numberOfColumns == 1);
+    attributes.shouldShowEditBelowContent = (self.numberOfColumns == 1);
+    attributes.shouldShowSmallThumbnailImage = (self.numberOfColumns == 1);
+    attributes.nodeNameHorizontalDisplacement = (self.numberOfColumns == 1)? 58 : 10;
+    attributes.nodeNameVerticalDisplacement = (self.numberOfColumns == 1)? 10 : self.thumbnailWidth + 20;
+    attributes.nodeNameFont = (self.numberOfColumns == 1)? [UIFont systemFontOfSize:17] : [UIFont fontWithName:@"HelveticaNeue-Light" size:13];
+    attributes.editImageTopSpace = (self.numberOfColumns == 1) ? 17 : 0;
```

Comment:
This big block of attributes seems to be copy/pasted three times which makes maintenance 3x as much work.